gdx/src/com/badlogic/gdx/graphics/glutils/GLFrameBuffer.java

```diff
@@ -126,6 +126,12 @@ public GLFrameBuffer (Pixmap.Format format, int width, int height, boolean hasDe
 	/** Override this method in a derived class to dispose the backing texture as you like. */
 	protected abstract void disposeColorTexture (T colorTexture);
 
+	protected void bindFrameBufferColorTexture () {
+		GL20 gl = Gdx.gl20;
+		gl.glFramebufferTexture2D(GL20.GL_FRAMEBUFFER, GL20.GL_COLOR_ATTACHMENT0, GL20.GL_TEXTURE_2D,
```

Comment:
Better move 2d texture specific code to the 2d texture implementation and make this abstract instead.